tslint.json

```diff
@@ -0,0 +1,27 @@
+{
+  "defaultSeverity": "error",
+  "extends": ["tslint:recommended", "tslint-config-prettier"],
+  "linterOptions": {
+    "exclude": [
+      "node_modules/*",
+      "src/serviceWorker.ts",
+      "src/setupTests.ts",
+      "src/react-app-env.d.ts"
```

Comment:
Do you want to add app.test here? I get a warning from that file when I lint.

Reply:
I think it'll get deleted so no need